corehq/apps/app_manager/templates/app_manager/profile.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version='1.0' encoding='UTF-8'?>
 <profile version="{{ app.version }}"
-         update="{{ profile_url }}"
+         update="{{ profile_url }}&amp;profile={{ build_profile }}"
```

Comment:
Does `{{ profile_url }}` already have a `?` param-string or should this `&` be a `?`?